epochStart/bootstrap/nodesCoordinator.go

```diff
@@ -10,22 +10,20 @@ import (
 	"github.com/ElrondNetwork/elrond-go/data/block"
 	"github.com/ElrondNetwork/elrond-go/data/state"
 	"github.com/ElrondNetwork/elrond-go/epochStart"
+	"github.com/ElrondNetwork/elrond-go/process"
 	"github.com/ElrondNetwork/elrond-go/sharding"
 )
 
 type nodesCoordinator struct {
```

Comment:
I suggest moving this in /sharding/nodescoordinator subpackage. It will be easier to maintain with other nodes coordinator implementations

Reply:
i want to keep it here for right now. as it is used inside - in next refactor we will see.